code/game/objects/items/storage/belt.dm

```diff
@@ -87,7 +87,6 @@
 /obj/item/storage/belt/utility/chief/full/PopulateContents()
 	new /obj/item/screwdriver/power(src)
 	new /obj/item/crowbar/power(src)
-	new /obj/item/weldingtool/experimental(src)//This can be changed if this is too much
```

Comment:
dunno if this is even spawned anywhere but I feel like this should just be replaced with a normal highcap welder probably

Reply:
I'm not sure it's used but yeah - must have missed it